### PR TITLE
Standardize LLaMA attention mask

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

**Comment by lugimzzz** [comment]
建议同步修改prefix tuning 中的pad_attention_mask

**Review by ZHUI — APPROVED** [review]
LGTM
